### PR TITLE
Edits to using_openshift/integrating_external_services as per card #544

### PR DESCRIPTION
**Pull request by bfallonf**

### CODE REVIEW

File: using_openshift/integrating_external_services.adoc [hunk 1]
```diff
@@ -11,43 +11,35 @@ toc::[]
 
 == Overview
 
-Many OpenShift applications use external resources (external databases, external SaaS endpoints,
-etc).  These external resources are simple to model as native OpenShift services so that 
-applications can work with them as they would any other internal service.
+Many OpenShift applications use external resources, such as an external databases, or an external SaaS endpoint. These external resources are can be modeled as native OpenShift services, so that applications can work with them as they would any other internal service.
```
Comment (adellape) [inline using_openshift/integrating_external_services.adoc]: s/are can/can/

File: using_openshift/integrating_external_services.adoc [hunk 2]
```diff
@@ -150,14 +138,11 @@ service:
...
-<1> Other fields on the `DeploymentConfig` are omitted
+<1> Other fields on the `DeploymentConfig` are omitted.
 
 === Using the service in your application
 
-Using an external service in your application is just like using an internal service.  Your
-application will be injected with environment variables for the service and the additional
-environment variables with the credentials described above.  In our example, The 'helloworld'
-container will receive the following environment variables:
+Using an external service in your application is just like using an internal service. Your application is injected with environment variables for the service and the additional environment variables with the credentials described above. In the following example, The 'helloworld' container receives the following environment variables:
```
Comment (adellape) [inline using_openshift/integrating_external_services.adoc]: s/The/the/

File: using_openshift/integrating_external_services.adoc [hunk 3]
```diff
@@ -11,43 +11,35 @@ toc::[]
...
+One of the most common types of external services is an external SaaS endpoint. As an example, assume that you want to use a SaaS provider for an application. This requires:
 
-1. An endpoint to communicate with
-2. A set of credentials:
-   a.  An API key
-   b.  A username
-   c.  A passphrase
+1. An endpoint to communicate with.
+2. A set of credentials, such as:
+   a.  An API key.
```
Comment (pmorie) [inline using_openshift/integrating_external_services.adoc]: Why use periods here?

Comment (bfallonf) [inline using_openshift/integrating_external_services.adoc]: @pmorie Yes, this one was incorrect. Apologies.

**Comment by bfallonf** [comment]
@pmorie I've fixed the periods in lists and put the scripts in example markup. I've also played with a sentence or two.

I am still waiting on the question I asked in trello. I'll copy it to here:

"I can see that this is outlining a procedure, and it states that it's a specific example (IE: Using an External SaaS Provider). I'm wondering how different the procedure would be if the reader wanted to use something different? For example, an external database. I'm just worried this section is too specific."

Thanks!

File: using_openshift/integrating_external_services.adoc [hunk 4]
```diff
@@ -150,24 +156,33 @@ service:
...
+
+SAAS_API_KEY=[replaceable]#<value of SaaS api key>#
+
+SAAS_USERNAME=[replaceable]#<value of SaaS username>#
+
+SAAS_PASSPHRASE=[replaceable]#<value of SaaS passphrase>#
+====
+
+<1> An example IP address.
+<2> An example port number.
```
Comment (adellape) [inline using_openshift/integrating_external_services.adoc]: @bfallonf If you meant for these to show up as a code block within an example, it's missing some `----`s (see https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#examples). Also the `<1>` and `<2>` should be within the same example block; right now they're just showing up as plain text.

**Comment by pmorie** [comment]
@bfallonf @adellape 

> "I can see that this is outlining a procedure, and it states that it's a specific example (IE: Using an External SaaS Provider). I'm wondering how different the procedure would be if the reader wanted to use something different? For example, an external database. I'm just worried this section is too specific."

It would be the exact same for an external database.  I can add an example for that if you would like.

**Comment by adellape** [comment]
@bfallonf We'll get @pmorie's external DB example (in https://github.com/openshift/openshift-docs/pull/194) added in after we get your PR here merged.

**Comment by bfallonf** [comment]
@adellape I put that into a block with ****'s around it. I think an example block is a bad idea because there are replaceables inside. I also got rid of the two footnote things and put them into the code block as <IP_Address> and <Port_Number>.

I'll also check out the other PR.

@adellape @pmorie Let me know if there's anything else for this PR.

**Comment by adellape** [comment]
This PR had some squashing issues, so closing it and continuing in https://github.com/openshift/openshift-docs/pull/196.

**Pull request closed without merging**